packages/medusa/src/api/admin/claims/[id]/inbound/items/[action_id]/route.ts

```diff
@@ -10,6 +10,7 @@ import {
   AuthenticatedMedusaRequest,
```

Comment:
question: Shouldn't we return the claim as well in all the `/claims` Rest APIs? It seems we are doing it in some but not all.